WooCommerce/WooCommerceTests/ViewRelated/Products/Actions Factory/ProductFormActionsFactoryTests.swift

```diff
@@ -87,6 +87,62 @@ final class ProductFormActionsFactoryTests: XCTestCase {
         XCTAssertEqual(factory.bottomSheetActions(), expectedBottomSheetActions)
     }
 
+    func test_viewModel_for_physical_simple_product_without_linked_products() {
```

Comment:
What do you think about describing the expectations too? 

```suggestion
    func test_viewModel_for_product_without_linked_products_shows_editLinkedProducts_in_bottom_sheet() {
```

I don't believe we need to mention “physical and simple” since the constraints seem to be only for the `upsellIDs` and `crossSellIDs`.